builtin.py

```diff
@@ -58,6 +59,34 @@ def advance(self, index: int) -> Node:
     def __str__(self):
         return self.title
 
+class Obj(Node):
+    def __init__(self, title:str, desc:str, player:Player):
+        super().__init__(title, desc)
+        self.desc = desc
+        self.player = player
+        if title.isnumeric(): #initializing money
+            self.amount_Money = int(title)
+        elif title[:2] == "po": #if first 2 characters are po then its a potion
+            if title[2:] == "health":
+                self.amount_Hp = 25
+                self.potionType = "Health potion"
+
+class Potion(Obj):
+    def on_select(self):
+        if self in self.player.bag:
+            if self.amount_Hp:
+                self.player.hp += self.amount_Hp
+                self.player.bag.remove(self)
```

Comment:
One interesting thing we could do, is take in a function that augments the player so we could have potions that do many things!

Reply:
Removed this functionality. Using a potion would be a separate function.